base/setup/reactos/lang/cs-CZ.rc

```diff
@@ -179,3 +182,29 @@ BEGIN
     IDS_PARTITION_SIZE "Size"
     IDS_PARTITION_STATUS "Status"
 END
+
+STRINGTABLE
+BEGIN
+    IDS_BOOTLOADER_NOINST    "No installation"
+    IDS_BOOTLOADER_REMOVABLE "Removable media"
+    IDS_BOOTLOADER_SYSTEM    "System partition (Default)" // For non-MBR disks
+    IDS_BOOTLOADER_MBRVBR    "MBR a VBR (Default)"        // For MBR disks only
+    IDS_BOOTLOADER_VBRONLY   "Jen VBR"                    // ""
```

Comment:
What is the purpose of that // "" comment? It doesn't tell my anything.

Reply:
It's the usual way to say that it's the same comment as above, without repeating it again.
https://en.wikipedia.org/wiki/Ditto_mark

Reply:
so what your comment wants to tell us is that IDS_BOOTLOADER_**VBRONLY** is // For MBR disks only ??? Sounds a bit weird, doesn't it?

Reply:
Let me **guess** what you actually want to say here, is that:
IDS_BOOTLOADER_SYSTEM is available only for GPT, and the default for GPT
IDS_BOOTLOADER_MBRVBR is available only for non-GPT, and is the default then
IDS_BOOTLOADER_VBRONLY is available only for non-GPT, but is not the default then

That could be written **once** somewhere in some .h or in some .cpp, where you think it would fit. But I don't see any reason why this should be written (in a slightly confusing way) into an rc, and then be duplicated into 38 other rc's as well. It will only confuse the translators. And is not necessary at all for them being able to translate those strings. I do recommend to strip all 3 comments from all rc's!

Reply:
Comments are for people writing code, going to these PRs and getting hung up about how a comment looks when you aren't contributing to the area is verbal bloat.

Reply:
Yes, exactly. And writing comments about obvious things, and then duplicating them 50 times, **does bloat the codebase**. If 99% of the github feedback deals about unneeded comments, then it might be worth considering commenting less about obvious things. Notice that people who avoid commenting the obvious do get immediate approval instead.

Reply:
Just because you don't understand it doesn't mean its useless

Reply:
I **do** understand that translators will not need those comments, and that the alignment of the comments will have to be updated recurrently, leading to translators having to touch more lines than necessary. And I **do** know that some translators will even start translating the comments unnecessarily.

Reply:
I don't see why you get to decide what translators need or not, I don't see why you get to decide what comments go into code at all.
Your obsession with what extra characters are in the code base make our jobs more difficult and you don't get that.

Reply:
Addressed, comments moved into resource.h